doc/releases/changelog-dev.md

```diff
@@ -92,32 +92,66 @@
       qml.RZ(omega, wires=wires[0])
       qml.GlobalPhase(-phase)
   
+  # This decomposition will be ignored for `QubitUnitary` on more than one wire.
   qml.add_decomps(QubitUnitary, zyz_decomposition)
   ```
-  
-  This decomposition will be ignored for `QubitUnitary` on more than one wire.
-
-* The :func:`~.transforms.decompose` transform now supports symbolic operators (e.g., `Adjoint` and `Controlled`) specified as strings in the `gate_set` argument
-  when the new graph-based decomposition system is enabled.
-  [(#7331)](https://github.com/PennyLaneAI/pennylane/pull/7331)
 
-  ```python
-  from functools import partial
-  import pennylane as qml
+* Symbolic operator types (e.g., `Adjoint`, `Controlled`, and `Pow`) can now be specified as strings
```

Comment:
This is a great changelog entry! 🎉

Reply:
@astralcai any idea why the formatting is appearing strangely? 🤔

Reply:
Strange how?

Reply:
<img width="490" alt="Screenshot 2025-05-05 at 9 40 33 AM" src="https://github.com/user-attachments/assets/b91e4cf5-f5d5-4f20-8124-c55e385e34bc" />

The back-ticked regions are orange 🤔

Reply:
![image](https://github.com/user-attachments/assets/fe60d821-af0c-4729-8ac0-528a9dffd444)
Seems to be rendering fine?

Reply:
Yeah weird... okay feel free to resolve this :)